### DISCUSSION TITLE
Add non-unique `updateMany`

### DISCUSSION THREAD

**Discussion by rachmanov** [keystonejs/keystone#8904] [Feature requests]
If I need to batch update data of certain list by some criteria, I need to create custom function to find IDs at first and then pass them to UpdateMany query.

Are there plans for the future to add such function in Keystone core?

**Comment by dcousens** [comment]
@rachmanov is `findMany` in some way [ineffective](https://keystonejs.com/docs/apis/query#find-many) for your usage?
